test/acceptance-tests/workspace-devtime/step-definitions/reading-artifacts/findModelProperty.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+var util = require('util');
+var async = require('async');
+var app = require('../../../../../');
+var loopback = require('loopback');
+var chai = require('chai');
+var path = require('path');
+var ModelProperty = app.models.ModelProperty;
+var exampleWorkspace = path.resolve(__dirname, '../../../../../example/common/models');
+
+module.exports = function() {
+  var testcase = this;
+  this.Given(/^The model '(.+)' has a property '(.+)'$/, function(modelName, propertyName, next) {
+    testcase.modelName = modelName;
+    testcase.propertyName = propertyName;
```

Comment:
Something is weird here, you get the modelNames and properties but don't use it down below.